### PR TITLE
Final: Capstone PR

### PR DESCRIPTION
**Pull request by wwwdrt**
# 🎯Learning objectives

- Understand how to use medium-fidelity wireframes to create a UI.
- Follow written requirements (e.g. user stories).
- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Use images and backgrounds to enhance the look of a website.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

# 🚀Overview

In this capstone project, you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course. It's very important you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.

# 📃General requirements

- [x] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- [x] Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

# 📃HTML/CSS & JavaScript requirements

- [x] Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

# 📃Project requirements

- [x] You should personalize the content of your page. Choose a topic that is different than the one in the original design.
- [x] You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
- [x] Colors.
- [x] Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
- [x] Layout: composition and space between elements.
- [x] The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- [x] You can use a CSS framework (for example Bootstrap) for styling, if you want to.
- [x] You should build only home and about pages.
- [x] Mobile: up to 768px wide.
- [x] Desktop: 768px or wider.
- [x] The home page should have a link in the menu to the about page.
- [x] The logo in the header links to the home page.
- [x] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [x] There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- [x] The mobile menu should have a close (X) button that closes the menu.
- [x] The section "Featured speakers" should be created dynamically in JavaScript.
- [x] You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
- [x] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

# 📃Project documentation

Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

- [x] A description of the project.
- [x] A demo of the project features (different pages and different screen sizes).
- [x] You should also highlight some interesting piece of code or something you built that you are very proud of.
- [x] For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.
- [x] Add the video link to your pull request description.

# 📝Note

👋Hey, I'm looking forward to your feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            <li class="md:w-[20%] lg:w-[28%]">
+                <a href="#">
+                    Logout
+                </a>
+            </li>
+        </ul>
+        <nav id="desktop-menu"
+             class="hidden md:flex items-center justify-center p-4 shadow-md fixed w-full top-0 left-0 bg-white mt-[2.2rem] z-[99]">
+            <a href="./pages/about.html">
+                <img src="./public/logo.png" alt="Logo" class="h-[53px] w-[81px] mr-8 cursor-pointer">
```
Comment (tufoinnkuo10) [inline index.html:45]: - [ ] great job. please like in the original  design, kindly align the logo with the headline text and position the nav links a little to the right. nice work
yours
Screenshot (1026)_LI
original
Screenshot (1026)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+            <span class="bar"></span>
+        </div>
+
+        <div id="menu-overlay" class="hidden md:hidden menu-overlay fixed" data-overlay>
+            <ul id="overlay"
+                class="flex flex-col gap-[3.5rem] pl-3 pt-3 text-corange text-5xl m-auto drop-shadow-2xl text-center">
+                <li>
+                    <a href="#hero" class="menu-link hover:btnEff-1">Home</a>
+                </li>
+                <li>
```
Comment (tufoinnkuo10) [inline index.html:83]: - [ ] please the Mobile home page does not have a logo on the headline section. it has thesame tiltles like in the desktop version. please follow the original design and fix that. 
yours
Screenshot (1033)
original
Screenshot (1034)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                    <br>
+                    2023</h2>
+                <div id="description" class="bg-herobox border-[6px] border-white p-[1rem] mt-[1.5rem] mb-[1.5rem]">
+                    <p>Welcome to the World Boxing Conference where
+                        champions are born and
+                        legacies
+                        are made. Join a global community of fighters and enthusiasts alike, and experience the electrifying rush of the sport that's captured hearts and minds around the world. Discover the power of the punch, the discipline of the training, and the camaraderie of the ring. Whether you're a seasoned pro or just starting out, the WBC is the ultimate destination for anyone looking to elevate their game and leave their mark on the world of boxing. Join us today and become part of the legacy.</p>
+                </div>
+                <h3 id="date" class="text-2xl font-bold text-date">2023.10.15(THU) ~ 16(FRI)</h3>
+                <p id="place" class="text-place mb-[6rem]">@ Sheikh Zayed Rd, Dubai World Trade Centre and more</p>
```
Comment (tufoinnkuo10) [inline index.html:116]: - [ ] your speakers section is not visible. please kindly fix that. both for the desktop and mobile version
Screenshot (1027)

File: pages/about.html [hunk 4]
```diff
@@ -121,7 +121,7 @@ <h2 class="pt-[3rem] text-2xl font-medium text-mainprogram">World Boxing Confere
             July</p>
         <div id="history-ad"
              class="border-2 border-navfont bg-white w-[90%] m-auto flex justify-center p-4 cursor-pointer">
-            <img src="../capstone/public/logo.png" alt="Logo" class="h-[104px] w-[193px] m-auto">
```
Comment (tufoinnkuo10) [inline pages/about.html:124]: - [ ] Grreat job with the about page . please kindly follow the design and personalize the content not add new design.please remove the video and make the logo to look like the original version.
yours
Screenshot (1029)_LI
original
Screenshot (1030)

File: pages/about.html [hunk 5]
```diff
@@ -163,11 +163,12 @@ <h2 class="text-5xl font-medium">2018</h2>
         <h2 class="pt-[3rem] text-2xl font-medium text-sponsors-font">Sponsors</h2>
         <hr class="hr">
```
Comment (tufoinnkuo10) [inline pages/about.html:164]: - [ ]  please reduce the size /height of the partner section of the mobile about page so it looks like the original design
yours
Screenshot (1031)
original
Screenshot (1032)

File: pages/about.html [hunk 6]
```diff
@@ -176,7 +177,7 @@ <h2 class="pt-[3rem] text-2xl font-medium text-sponsors-font">Sponsors</h2>
 <section id="footer"
          class="md:bg-footer flex justify-center items-center text-start m-auto gap-4 md:text-white text-mainprogram">
     <figure>
-        <img src="../capstone/public/logo.png" alt="Logo" class="p-8">
+        <img src="../public/logo.png" alt="Logo" class="p-8">
     </figure>
     <div id="footer-info" class="flex flex-col gap-4 w-[80%] p-8">
```
Comment (tufoinnkuo10) [inline pages/about.html:182]: - [ ] please reduce the height of the about desktop footer so it can match the original design
Screenshot (1028)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @wwwdrt 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ design well followed

video presentation
Great job with the code presentation. please always give a nice reason why you build what you built and explanations when presenting your work rather than just clicking through. it will help some viewers understand more. nice work

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 3 above)
Comment (wwwdrt) [reply index.html:116]: This one is alright, GitHub pages is very slow 🫡

File: index.html (hunk 1 above)
Comment (wwwdrt) [reply index.html:45]: 👉✅

image

File: index.html (hunk 2 above)
Comment (wwwdrt) [reply index.html:83]: 👉✅
image

File: pages/about.html (hunk 4 above)
Comment (wwwdrt) [reply pages/about.html:124]: 👉✅

image

File: pages/about.html (hunk 5 above)
Comment (wwwdrt) [reply pages/about.html:164]: 👉✅

image

File: pages/about.html (hunk 6 above)
Comment (wwwdrt) [reply pages/about.html:182]: 👉✅

image

**Review by emmiiorji — APPROVED** [review]
Hi @wwwdrt, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**